tests-integration/tests/primtive.rs

```diff
@@ -7,7 +7,6 @@ use andromeda_primitive::mock::{
     mock_store_value_msg,
```

Comment:
I just realised this script is called `primtive` instead of `primitive`, should probably fix this